kernel/classes/ezcontentobjecttreenodeoperations.php

```diff
@@ -113,6 +113,7 @@ static function move( $nodeID, $newParentNodeID )
                 $nodeIDList = array( $nodeID );
                 eZSearch::removeNodeAssignment( $node->attribute( 'main_node_id' ), $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList );
                 eZSearch::addNodeAssignment( $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList, true );
+                eZSearch::refreshChildren( $node );
```

Comment:
@andrerom I think that both scripts need to be active. The from legacy to handle standard delayed indexing actions, and the from eZ Platform Search, to handle changing the visibility of a node.

Look here: https://github.com/netgen/ezplatformsearch/pull/3

Ping @harmstyler 

I would like to see them merged into one somehow, however.